cmd/oceantv/broadcast_hardware_machine.go

```diff
@@ -32,6 +32,16 @@ func newHardwareStarting(ctx *broadcastContext) *hardwareStarting {
 }
 func (s *hardwareStarting) enter() {
 	s.LastEntered = time.Now()
+	if s.cfg.ControllerMAC != 0 {
+		controllerIsOn, err := getDeviceStatus(context.Background(), s.cfg.ControllerMAC, settingsStore)
+		if err != nil {
```

Comment:
Maybe we should just combine these cases. Also, we should probably perform a notification under this circumstance e.g. 
```
if err != nil || !controllerIsOn {
    notifier.Send(
        context.Background(),
        _cfg.SKey,
        "health",
        fmt.Sprintf("Could not get good controller status on hardware startup, status: %v, err: %v",controllerIsOn,err),
    )
    s.bus.publish(controllerFailureEvent{})
    return
}
```

Reply:
I think that the 2 cases represent different things though? One is a data store error (I think) and the other is a hardware error. Would probably be good to know if we're having data store errors if we are?

Reply:
Your handling for both of these cases was the same i.e. to publish a controllerFailureEvent, which I think is fair. 

The suggested code provides the error in the notification message, so if the controllerFailureEvent occurred because of a datastore error, we can be informed of that.

Reply:
The controller failure event handling will already notify (through the onfailureclosure call). Is there a good way of handling the error through this event to make a datastore error visible to the notifier to send?

Reply:
I don't think it matters too much if we notify in both places, but that raises a good question; maybe it'd be better to store a failure message in the event. That way we can get that and use it for the notify of the handling of the event. 

Yep, it's in the code I provided :)

Reply:
Haven't we tried to avoid over notifying recently? I think I'm more inclined at this point to not differentiate between a datastore error and controller not reporting, as we will probably be able to tell when we check the voltage after receiving the notification.